### ISSUE TITLE
lint: sanity tests fail when modules are enabled

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#627]
Execute:
```bash
export GO111MODULE=on
go test -v -count=1 github.com/go-critic/go-critic/lint
```
Result:
```
    --- FAIL: TestSanity/unlambda (0.01s)
        all_checkers_test.go:258: github.com/go-critic/go-critic/lint/testdata//_sanity package is not properly loaded
    --- FAIL: TestSanity/unnamedResult (0.01s)
        all_checkers_test.go:258: github.com/go-critic/go-critic/lint/testdata//_sanity package is not properly loaded
    --- FAIL: TestSanity/unnecessaryBlock (0.01s)
        all_checkers_test.go:258: github.com/go-critic/go-critic/lint/testdata//_sanity package is not properly loaded
    --- FAIL: TestSanity/unslice (0.01s)
        all_checkers_test.go:258: github.com/go-critic/go-critic/lint/testdata//_sanity package is not properly loaded
    --- FAIL: TestSanity/unusedParam (0.01s)
        all_checkers_test.go:258: github.com/go-critic/go-critic/lint/testdata//_sanity package is not properly loaded
    --- FAIL: TestSanity/yodaStyleExpr (0.01s)
        all_checkers_test.go:258: github.com/go-critic/go-critic/lint/testdata//_sanity package is not properly loaded
```
Expected result: tests are passing.

**Comment by cristaloleg** [comment]
Can you reproduce it every time? 🤔

**Comment by Quasilyte** [comment]
Will be fixed as https://github.com/go-lintpack/lintpack/pull/63 gets merged.
